### PR TITLE
[rte] Make min-height of editor changeable.

### PR DESCRIPTION
**Pull request by rabengraph**
https://deploy-preview-475--comet-admin.netlify.app/?path=/story/comet-admin-rte--rte-theming

### CODE REVIEW

File: packages/admin-rte/src/core/Rte.tsx [hunk 1]
```diff
@@ -86,13 +86,21 @@ export interface CometAdminRteThemeProps {
...
+    };
 }
 
-export interface IProps extends CometAdminRteThemeProps {
+export interface IProps {
     value: EditorState;
     onChange: OnEditorStateChangeFn;
     options?: IOptions;
     disabled?: boolean;
+    theme?: CometAdminRteThemeProps;
```
Comment (rabengraph) [inline packages/admin-rte/src/core/Rte.tsx:99]: I put all theme-props  into a separate property, to improve the structure.

File: packages/admin-rte/src/core/Rte.tsx [hunk 2]
```diff
@@ -86,13 +86,21 @@ export interface CometAdminRteThemeProps {
...
+    /**
+     * @deprecated Use theme.colors instead.
+     */
+    colors?: any;
```
Comment (rabengraph) [inline packages/admin-rte/src/core/Rte.tsx:103]: Passing colors didn't work somehow (at least I couldn't get it to work), so I do not handle the deprecated-case as I think it's broken anyway.

Comment (nsams) [reply packages/admin-rte/src/core/Rte.tsx:103]: I think we can/should remove this depreacted prop for comet-admin 2

**Review by jamesricky — APPROVED** [review]

**Review by jamesricky — APPROVED** [review]

File: packages/admin-stories/src/admin-rte/RteTheming.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    return (
+        <>
+            <Box marginBottom={4}>
+                <Card variant="outlined">
+                    <CardContent>
+                        <Rte
+                            value={editorState}
+                            onChange={setEditorState}
+                            theme={{ editor: { minHeight: select("minHeight", [0, 100, 200, 300, 1000], 0) } }}
```
Comment (nsams) [inline packages/admin-stories/src/admin-rte/RteTheming.tsx:20]: I think changing the minHeight is a common property to change, so I propose to expose it like `<Rte minHeight={100} />`.

File: packages/admin-rte/src/core/Rte.tsx (hunk 1 above)
Comment (nsams) [reply packages/admin-rte/src/core/Rte.tsx:99]: How is this implemented for other components: do we have a way to customize theme (colors etc) using props?

If not, does it really make sense to have it for this component?

Comment (rabengraph) [reply packages/admin-rte/src/core/Rte.tsx:99]: It's probably really better to directly change the minHeight without involving the theme, for the sake of simplicity. 

> do we have a way to customize theme (colors etc) using props?

No, this does not exist.

File: packages/admin-stories/src/admin-rte/RteTheming.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+                    <CardContent>
+                        <Rte value={editorState} onChange={setEditorState} minHeight={select("minHeight", [0, 100, 200, 300, 1000], 0)} />
+                    </CardContent>
+                </Card>
+            </Box>
+        </>
+    );
+}
+
+storiesOf("@comet/admin-rte", module).add("Rte, theming", () => <Story />);
```
Comment (nsams) [inline packages/admin-stories/src/admin-rte/RteTheming.tsx:25]:
```suggestion
storiesOf("@comet/admin-rte", module).add("Rte, minHeight", () => <Story />);
```

**Review by nsams — APPROVED** [review]

**Pull request merged**